pkg/platform/runtime/buildexpression/merge/merge_test.go

```diff
@@ -108,6 +107,7 @@ runtime = solve(
 	],
 	requirements = [
 		Req(name = "language/perl"),
+		Req(name = "language/perl/JSON"),
```

Comment:
Yes, it fails with:
```
Unable to update buildexpression with merge results:
    Could not update BuildExpression's requirements: Could not remove requirement '[ACTIONABLE]JSON[/RESET]', because it does not exist.
```

I can revert the changes, skip the test, and file a follow up if you feel this is a legitimate bug. This is another one I meant to come back to but with the sheer number of changes I missed it. Thanks for catching.

Reply:
Thanks for double-checking. You can leave it as-is.